userspace/libscap/scap_procs.c

```diff
@@ -40,6 +40,10 @@ limitations under the License.
 #include "windows_hal.h"
 #endif
 
+#if defined(_WIN64) || defined(WIN64) || defined(_WIN32) || defined(WIN32)
```

Comment:
can we use `#ifdef SYSDIG_WIN` here? if not, maybe move the `#define` to somewhere we'll pick it up? otherwise you're defining it for just a single use and we might as well drop it

Reply:
So for this, I was thinking about a more comprehensive fix at a later date. We need to find all platform related defines (I see there's _MSC_VER is a few places and sun too). Would be good to have a central place where we look for all platform specific defines and create defines like SYSDIG_WIN, SYSDIG_SUN, etc and use them consistently. Right now, I don't see a consistent pattern. I would like to have a more complete definition of SYSDIG_WIN and then move it to another header from where we can pick it up and use everywhere.

Reply:
okay, so in that case, I'd drop the `SYSDIG_WIN` macro altogether and get back to this later.

Reply:
Done. I've removed SYSDIG_WIN.